docs/product-user-guide/src/main/asciidoc/chap-process-designer.adoc

```diff
@@ -2094,7 +2077,7 @@ Directories with work items must contain:
 * A work item configuration file.
 * All resources referenced in the work item configuration file: icon, documentation, and dependencies.
 
-A _work item configuration file_ is a file with the same name as the parent directory, for example `Twitter.wid`, that contains details about the work item resources in the service repository. The file is an extension of the work item definition file (see <<_sect_work_item_definition>>). Note that the configuration file must contain references to any dependencies the work item handler requires. Optionally, it can define the documentation property with a path to documentation and category which defines the category the custom work item is placed under in the repository. 
+A _work item configuration file_ is a file with the same name as the parent directory, for example `Twitter.wid`, that contains details about the work item resources in the service repository. The file is an extension of the work item definition file (see <<_sect_work_item_definition>>). Note that the configuration file must contain references to any dependencies the work item handler requires. Optionally, it can define the documentation property with a path to documentation and category which defines the category the custom work item is placed under in the repository.
```

Comment:
".......category which defines the category the custom work item is placed under in the repository."
I think there is a missing word here. Can you rephrase?